### PR TITLE
Second task

### PR DESCRIPTION
**Pull request by YANGMAO-ZHANG**
@Felixnoo 
#1854

### CODE REVIEW

File: content/zh/docs/installing-on-linux/public-cloud/install-kubesphere-on-azure-vms.md [hunk 1]
```diff
@@ -253,14 +254,18 @@ Azure Virtual Network doesn't support the IPIP mode used by [Calico](https://doc
...
+由于 Kubernetes 集群直接设置在 Azure 实例上，因此负载均衡器未与 [Kubernetes 服务](https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer)集成。但是，您仍然可以手动将 NodePort 映射到负载均衡器。这需要两个步骤：
 
-As the Kubernetes cluster is set up on Azure instances directly, the load balancer is not integrated with [Kubernetes Services](https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer). However, you can still manually map the NodePort to the load balancer. There are 2 steps required.
+1. 在负载均衡器中创建新的负载均衡规则
 
-1. Create a new Load Balance Rule in the load balancer.
    ![Load Balancer](/images/docs/aks/azure-vm-loadbalancer-rule.png)
-2. Create an Inbound Security rule to allow Internet access in the Network Security Group.
+
+2. 创建 Inbound Security 规则以允许网络安全组中的 Internet 访问。
```
Comment (Felixnoo) [inline content/zh/docs/installing-on-linux/public-cloud/install-kubesphere-on-azure-vms.md:268]: `创建 Inbound Security 规则以允许网络安全组中的 Internet 访问。` 
->
`在网络安全组中创建入站安全规则以允许外网访问。`

**Review by Felixnoo — CHANGES_REQUESTED** [review]
Please take a look at my comments, thanks.

File: content/zh/docs/installing-on-linux/public-cloud/install-kubesphere-on-azure-vms.md [hunk 2]
```diff
@@ -132,32 +133,32 @@ After you download KubeKey, if you transfer it to a new machine also with poor n
...
-   Make `kk` executable:
+   给予 `kk` 执行权限:
 
    ```bash
-   chmod +x kk
+chmod +x kk
    ```
 
-2. Create an example configuration file with default configurations. Here Kubernetes v1.20.4 is used as an example.
+2. 使用默认配置创建示例配置文件，这里以 Kubernetes v1.20.4 为例。
```
Comment (Felixnoo) [inline content/zh/docs/installing-on-linux/public-cloud/install-kubesphere-on-azure-vms.md:146]: Fix the format here.

**Comment by YANGMAO-ZHANG** [comment]
Thanks for your time. I have recovered the wrong version from Git, so this is not the latest version of my translation. Really apologize for wasting your time.  I have fixed all the bugs you mentioned, and I would like to review the file again. Again, really apologize, and thanks for your patient.

**Comment by Felixnoo** [comment]
> Thanks for your time. I have recovered the wrong version from Git, so this is not the latest version of my translation. Really apologize for wasting your time. I have fixed all the bugs you mentioned, and I would like to review the file again. Again, really apologize, and thanks for your patient.

No worries. Bugs are meant to be fixed by our joint efforts. Take your time.

File: content/zh/docs/installing-on-linux/public-cloud/install-kubesphere-on-azure-vms.md [hunk 3]
```diff
@@ -4,81 +4,82 @@ keywords: "KubeSphere, Installation, HA, high availability, load balancer, Azure
...
 linkTitle: "在 Azure VM 实例上部署 KubeSphere"
 Weight: 3410
+
 ---
 
-Using the [Azure cloud platform](https://azure.microsoft.com/en-us/overview/what-is-azure/), you can either install and manage Kubernetes by yourself or adopt a managed Kubernetes solution. If you want to use a fully-managed platform solution, see [Deploy KubeSphere on AKS](../../../installing-on-kubernetes/hosted-kubernetes/install-kubesphere-on-aks/) for more details. 
+您可以使用 [Azure 云平台](https://azure.microsoft.com/zh-cn/overview/what-is-azure/)自行安装和管理 Kubernetes，或采用托管 Kubernetes 解决方案。如果要使用完全托管平台解决方案，请参阅 [在 AKS 上部署 KubeSphere](../../../installing-on-kubernetes/hosted-kubernetes/install-kubesphere-on-aks/) 。
 
-Alternatively, you can set up a highly-available cluster on Azure instances. This tutorial demonstrates how to create a production-ready Kubernetes and KubeSphere cluster.
+此外，您也可以在 Azure 实例上搭建高可用集群。本指南演示如何创建生产就绪的 Kubernetes 和 KubeSphere 群集。
```
Comment (Felixnoo) [inline content/zh/docs/installing-on-linux/public-cloud/install-kubesphere-on-azure-vms.md:12]: `群集` -> `集群`

File: content/zh/docs/installing-on-linux/public-cloud/install-kubesphere-on-azure-vms.md [hunk 4]
```diff
@@ -4,81 +4,82 @@ keywords: "KubeSphere, Installation, HA, high availability, load balancer, Azure
...
-| ---------- | -------- | -------------- | ------------ | -------------------------------- | ------------ |
-| ssh        | TCP      | Inbound NAT    | 22           | 50200, 50201, 50202, 50100~50199 | Master, Node |
-| apiserver  | TCP      | Load Balancing | 6443         | 6443                             | Master       |
-| ks-console | TCP      | Load Balancing | 30880        | 30880                            | Master       |
-| http       | TCP      | Load Balancing | 80           | 80                               | Node         |
-| https      | TCP      | Load Balancing | 443          | 443                              | Node         |
+| 服务                    | 协议                   | 规则                           | 后端端口              | 前端端口                                            | 节点池           |
+| ----------------------- | ---------------------- | ------------------------------ | --------------------- | --------------------------------------------------- | ---------------- |
+| ssh <img width="60px"/> | TCP<img width="20px"/> | Inbound NAT<img width="30px"/> | 22<img width="60px"/> | 50200, 50201, 50202, 50100~50199<img width="10px"/> | 主节点, 普通节点 |
```
Comment (Felixnoo) [inline content/zh/docs/installing-on-linux/public-cloud/install-kubesphere-on-azure-vms.md:42]: Delete all the `` in this row.

Comment (Felixnoo) [inline content/zh/docs/installing-on-linux/public-cloud/install-kubesphere-on-azure-vms.md:42]: `Inbound NAT` -> `入站 NAT`

File: content/zh/docs/installing-on-linux/public-cloud/install-kubesphere-on-azure-vms.md [hunk 5]
```diff
@@ -4,81 +4,82 @@ keywords: "KubeSphere, Installation, HA, high availability, load balancer, Azure
 description: "了解如何在 Azure 虚拟机上创建高可用 KubeSphere 集群。"
 linkTitle: "在 Azure VM 实例上部署 KubeSphere"
 Weight: 3410
+
 ---
 
-Using the [Azure cloud platform](https://azure.microsoft.com/en-us/overview/what-is-azure/), you can either install and manage Kubernetes by yourself or adopt a managed Kubernetes solution. If you want to use a fully-managed platform solution, see [Deploy KubeSphere on AKS](../../../installing-on-kubernetes/hosted-kubernetes/install-kubesphere-on-aks/) for more details. 
+您可以使用 [Azure 云平台](https://azure.microsoft.com/zh-cn/overview/what-is-azure/)自行安装和管理 Kubernetes，或采用托管 Kubernetes 解决方案。如果要使用完全托管平台解决方案，请参阅 [在 AKS 上部署 KubeSphere](../../../installing-on-kubernetes/hosted-kubernetes/install-kubesphere-on-aks/) 。
```
Comment (Felixnoo) [inline content/zh/docs/installing-on-linux/public-cloud/install-kubesphere-on-azure-vms.md:10]: Please delete the extra spaces between `参阅` and `在` / `KubeSphere` and `。`

**Review by Felixnoo — CHANGES_REQUESTED** [review]

**Pull request merged**